src/components/10-atoms/button/index.js

```diff
@@ -10,9 +10,12 @@ import buttonCSS from './index.scss';
 
 const ARROW_RIGHT = 'arrow-right';
 
+// eslint-disable-next-line no-undef
+const isNativeShadowDOM = ShadowRoot.toString().indexOf('native code') > -1;
+
 // @TODO: REMOVE ONCE IE11 is deprecated!!!!
 // equivalent to event.isTrusted. Unfortunately, IE11 does not support it
-const eventIsNotTrusted = e =>
+const eventIsTrusted = e =>
   !(!e.screenX && !e.screenY && !e.clientX && !e.clientY);
```

Comment:
According to deMorgan's law, this can be greatly simplified to `e.screenX || e.screenY && e.clientX || e.clientY`

Reply:
fixed